src/content/en/fundamentals/media/mse/basics.md

```diff
@@ -91,6 +91,11 @@ Here, in no particular order, are a few things I won't cover.
    `<audio>` and `<video>` elements.
 +  Error handling for the sake of simplicity and clarity.
 
+Warning: Handling error events, API exceptions, and proactively checking
+`HTMLMediaElement readyState` and `MediaSource readyState` values before making
+calls on their APIs are mandatory in a production usage of the APIs as these
+values can change before associated events are delivered.
+
```

Comment:
"mandatory" makes it seem like the API won't work without this. It will, in most cases. Just wondering if there's precedent for such a strong wording (I wish devs *would* always do the error checking, etc.) -- would "strongly recommended" be better?

Also, when thinking about related MSE API usage problems, frequently web authors new to MSE make mistakes around the SourceBuffer.updating state. Perhaps mention that too? (At minimum, exception handling would help them know when they're making wrong assumptions about updating state, so maybe this is too much detail for this intro.)